libs/community/tests/unit_tests/utilities/test_yandex_search.py

```diff
@@ -0,0 +1,128 @@
+import unittest
+
+import responses
+
+from langchain_community.utilities.yandex_search import YandexSearchAPIWrapper
+
+
+class TestYandexSearchAPIWrapper(unittest.TestCase):
```

Comment:
We use the pytest framework for unit tests. Do you mind swapping over for consistency with other langchain code? It usually results in much less code as well

Reply:
Done